docs/semantics-of-k.tex

```diff
@@ -2,13 +2,13 @@
 
   % Package for using the full page.
   \usepackage{fullpage}
-  
+
```

Comment:
I would have put the semantics-of-k files into a different pull request.

Reply:
Oups! UI thought I did `git commit -a` .... :-(
I'll make sure to remove those, as they were partial anyway, and some I think were done by my editor automatically.